me.sergiotarxz.Exd.yml

```diff
@@ -0,0 +1,120 @@
+          commands:
+            # Have Flatpak run the GNU-compatible configure script.
+            - 'ln -s configure{.gnu,}'
+      # Restore write permission to the Perl libraries.
+    post-install:
+        - 'chmod -R u+w /app/lib/perl5'
+      # Clean up a bunch of stuff we don't need. Depending on your application,
+      # you may have to drop some of these (e.g. *.pod).
+    cleanup:
+        - '/bin/corelist'
+        - '/bin/cpan'
+        - '/bin/enc2xs'
+        - '/bin/encguess'
+        - '/bin/h2ph'
+        - '/bin/h2xs'
+        - '/bin/instmodsh'
+        - '/bin/json_pp'
+        - '/bin/libnetcfg'
+        - '/bin/perlbug'
+        - '/bin/perldoc'
+        - '/bin/perlivp'
+        - '/bin/perlthanks'
+        - '/bin/piconv'
+        - '/bin/pl2pm'
+        - '/bin/pod*'
+        - '/bin/prove'
+        - '/bin/ptar*'
+        - '/bin/shasum'
+        - '/bin/splain'
+        - '/bin/xsubpp'
+        - '/bin/zipdetails'
+        - '/include'
+        - '/man'
+        - '*.pod'
+
+  # Installs the modules generated by flatpak-cpan-generator.
+  # Note that *any* Makefile.PL-style modules MUST be installed in this one step,
+  # as once perllocal.pod is written in one module, it cannot be modified by others.
+  - name: perl-libs
+    buildsystem: simple
+    build-commands:
+      - 'perl-libs/install.sh'
+    # Same as with the Perl module, we need to restore write permission.
+    # However, -f is now passed to avoid errors from trying to touch files from the
+    # above module that are now marked as read-only.
+    post-install:
+      - 'chmod -Rf u+w /app/lib/perl5/site_perl'
+    sources:
+      - generated-sources.json
+    # This step should be customized based on the CPAN packages you're using.
+    cleanup:
+      - '/bin'
+      - '/man'
+  - name: "Hiperthermia"
+    buildsystem: simple
+    build-commands:
+      - 'cd HiperthermiaSource; perl lib/Exd/Fontconfig.pm'
+      - 'cd HiperthermiaSource; perl lib/Exd/FileFormat/Fontconfig.pm'
+      - 'cd HiperthermiaSource; perl -Ilib lib/Exd/Gui.pm'
+      - 'cd HiperthermiaSource; chmod 755 -R _Inline/'
+      - 'cp HiperthermiaSource/app_config.prod.json HiperthermiaSource/app_config.json'
+      - 'cp -vr HiperthermiaSource ${FLATPAK_DEST}/Hiperthermia'
+      - 'glib-compile-resources --sourcedir=${FLATPAK_DEST}/Hiperthermia ${FLATPAK_DEST}/Hiperthermia/resources.xml'
+      - 'install -Dm644 HiperthermiaSource/me.sergiotarxz.Exd.desktop /app/share/applications/me.sergiotarxz.Exd.desktop'
+      - 'install -Dm644  HiperthermiaSource/me.sergiotarxz.Exd.mime.xml /app/share/mime/packages/me.sergiotarxz.Exd.mime.xml'
+      - 'install -Dm755 HiperthermiaSource/run.pl /app/bin/hiperthermia'
+      - 'install -Dm644 HiperthermiaSource/exd-logo.png /app/share/icons/hicolor/256x256/apps/me.sergiotarxz.Exd.png'
+      - 'update-mime-database /app/share/mime'
+      - 'install -Dm644 HiperthermiaSource/me.sergiotarxz.Exd.metainfo.xml -t /app/share/metainfo/' 
```

Comment:
this should be done by a buildsystem.

Reply:
I am using Perl, a build system which does this does not exist for Perl.

Reply:
using Perl doesn't prevent using a build system like autotools, meson or cmake.